.github/workflows/tag_release.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         uses: svenstaro/upload-release-action@v2
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: ${{github.workspace}}/sonar-ios-plugin/target/*.jar
+          file: ${{github.workspace}}/lib/ecocode-sonar-ios-plugin-${{ github.ref_name }}.jar
```

Comment:
why change `target`to `lib` ? this modification implies that we must use `mvn package` (whihch produces jar file in `lib` directory) instead of `mvn compile` (whihch produces jar file in `target` directory) ... this change usage of tools scripts.

Reply:
I fixed if in my previous commit (found the problem myself).
I did that because at first I did not redirect 'mvn package' output to /lib.